src/components/CytoscapeGraph/graphs/GraphBadge.ts

```diff
@@ -127,3 +128,9 @@ export class RouteRuleBadge extends GraphBadge {
     super(ROUTE_BADGE, PfColors.Purple300, 'top');
   }
 }
+
+export class MissingSidecarsBadge extends GraphBadge {
+  constructor() {
+    super(SIDECAR_BADGE, PfColors.Red100, 'bottom');
```

Comment:
I'm not UX but I would suggest keeping the position the same as the other badges, otherwise it gets noisy and badges float around all around the node.  Also, I think I prefer keeping the colors the same, but then again, this badge indicates a problem, so perhaps red is OK.

Reply:
@serenamarie125 ?

Reply:
@jshaughn Yes, we need some UX feedback here.
I think to fix badge positioning we will need a Jira, because only fixed positioning is available now. With the current code, they will look too far if there aren't any other badges or with a space in between if it's only the CI visible:
![image](https://user-images.githubusercontent.com/23639005/40016506-25d7fae0-577c-11e8-97bf-8aad752dd673.png)

I can use this positioning for now if you think it's better while we work with UX in a new Jira.

Reply:
@israel-hdez Maybe someone changed something, After adding the route rule badge I thought I had it such that badges would pull left, and then line up to the right.  I don't care too much for the icons/badging to be honest.

Reply:
@jshaughn I'll leave the positioning as is for now... We could think about changing it when it becomes a problem due the amount of badges or when we have UX feedback.